interviews/frontend-takehome.md

```diff
@@ -35,9 +35,9 @@ Application can be used smoothly and without visible bugs.
 
 Non-core features encouraged for completion after the core features.
 
-- 💖 +2: Requested delighter content completed fully and Solution Completion is 💖4
-- 👍 +1: Requested delighter completed with bugs, or completed well but Solution Completion is 👍3
-- 👎 0: No delighter content was completed, or Solution Completion is 🛑1 or 👎2
+- 💖 +2: Requested delighter content completed fully
```

Comment:
Lol I just realized that these are supposed to map to the greenhouse rubric - i never made that connection before and was always a little confused on how to put translate the delighter to it.

Can we switch the hearts to stars too, since that's what it is in greenhouse?

Reply:
UUUUUUUUUGGGHHHHHH the sparkly hearts represent our culture so much better 😢 😭 😿 

...but yes, @borisonr and I were just talking about this on Slack haha. I'll send that too.

Reply:
time to reach out to greenhouse to request new emojis?